File: text_extraction.ts
import { ComputerVisionClient } from '@azure/cognitiveservices-computervision';
import { ApiKeyCredentials } from '@azure/ms-rest-js';

const credentials = new ApiKeyCredentials({ inHeader: { 'Ocp-Apim-Subscription-Key': 'your-key' } });
const client = new ComputerVisionClient(credentials, 'your-endpoint');

const imageData = null; // Replace with your image data source 

File: text_extraction.py
from azure.cognitiveservices.vision.computervision import ComputerVisionClient
from msrest.authentication import CognitiveServicesCredentials

# Initialize the client
credentials = CognitiveServicesCredentials('your-key')
client = ComputerVisionClient(credentials, 'your-endpoint')

# For image_data, prepare your image
image_data = None  # Replace with your image data source 